tools/chapel-py/src/core-types-gen.cpp

```diff
@@ -92,7 +97,9 @@ struct InvokeHelper<void(Args...)> {
   template <typename F>
   static PyObject* invoke(ContextObject* contextObject, F&& fn) {
     fn();
-    Py_RETURN_NONE;
+    // In python3.12, Py_None is immortal and this is not needed
+    Py_INCREF(Py_None);
+    return Py_None;
```

Comment:
I would've expected `Py_RETURN_NONE` to take care of increfs (or skipping them for immortal objects) etc. Is it not ABI-stable?

Reply:
Its not ABI stable sadly